packages/cli-kit/src/public/node/themes/factories.ts

```diff
@@ -56,18 +56,22 @@ export function buildBulkUploadResults(
   bulkUploadResponse: RemoteBulkUploadResponse[],
   assets: AssetParams[],
 ): Result[] {
-  const results: Result[] = []
-  if (!bulkUploadResponse) return results
+  if (bulkUploadResponse.length === 0 && assets.length !== 0) {
```

Comment:
I believe that the server will either return an array of `RemoteBulkUploadResponse` with the same length as `assets`, or it will return `[]`

I don't believe that we will encounter the case where `bulkUploadResponse.length > 0 && bulkUploadResponse.length < assets.length)